en/docs/chapter_sorting/selection_sort.md

```diff
@@ -1,14 +1,14 @@
 # Selection sort
 
-<u>Selection sort</u> works on a very simple principle: it starts a loop where each iteration selects the smallest element from the unsorted interval and moves it to the end of the sorted interval.
+<u>Selection sort</u> works on a very simple principle: it uses a loop where each iteration selects the smallest element from the unsorted interval and moves it to the end of the sorted section.
 
-Suppose the length of the array is $n$, the algorithm flow of selection sort is as shown in the figure below.
+Suppose the length of the array is $n$, the steps of selection sort is shown in the figure below.
 
 1. Initially, all elements are unsorted, i.e., the unsorted (index) interval is $[0, n-1]$.
 2. Select the smallest element in the interval $[0, n-1]$ and swap it with the element at index $0$. After this, the first element of the array is sorted.
 3. Select the smallest element in the interval $[1, n-1]$ and swap it with the element at index $1$. After this, the first two elements of the array are sorted.
 4. Continue in this manner. After $n - 1$ rounds of selection and swapping, the first $n - 1$ elements are sorted.
-5. The only remaining element is necessarily the largest element and does not need sorting, thus the array is sorted.
+5. The only remaining element is subsequently the largest element and does not need sorting, thus the array is sorted.
```

Comment:
`Necessarily` accurately conveys this inevitability. It emphasizes that, by the algorithm's logic, the last remaining element cannot be anything other than the largest. But the `subsequently` doesn't capture the inevitability that the last element must be the largest.

Maybe we can use: The only remaining element is necessarily the largest and does not require sorting, thus the array is sorted.

Reply:
I think "subsequently" here is trying to emphasize the causal relationship between step 1 ~ 4 to the outcome of step 5. This is strictly a difference in perspective so I still think "subsequently" is better.